src/controllers/update/beneficial.owner.type.controller.ts

```diff
@@ -54,3 +64,29 @@ const getNextPage = (beneficialOwnerTypeChoices: BeneficialOwnerTypeChoice | Man
         return config.UPDATE_BENEFICIAL_OWNER_INDIVIDUAL_URL;
   }
 };
+
+const retrieveBeneficialOwners = async (req: Request, appData: ApplicationData) => {
+  const session = req.session as Session;
+  const pscs = await getCompanyPsc(req, appData[EntityNumberKey] as string);
+  const raw = pscs as any;
+  if (pscs !== undefined) {
+    for (const item of (raw.items || [])) { //  kind does not exist on getCompanyPsc return type
```

Comment:
maybe move comment to line 71.

Reply:
I think items is defined on pscs so no need to use raw here (for loop).

Reply:
Yes items is present on pscs, but I couldn't access kind on it. Raw has kind but needs to be looped to access kind. 

I was trying to avoid another loop hence decided to use raw for the loop

Reply:
can do a  "const raw = item as any" in loop